src/main/java/com/taskmaster/server/projects/ProjectsDto.java

```diff
@@ -0,0 +1,5 @@
+package com.taskmaster.server.projects;
+
+public record ProjectsDto(Long id, String name) {
```

Comment:
Do we expect to receive a project with just a named? You are checking in the service class if the id is null, why not use "long" primitive here and remove the check?